fragments/install/cephadm/configure_client.yaml

```diff
@@ -0,0 +1,15 @@
+tests:
```

Comment:
We have `test_cephadm_add_client.yaml` fragment, can we override it instead of creating a new one?

Reply:
Sure pragdeesh...